File: 2DAM/Programacion multimedia y dispositivos moviles/Primer trimestre/Tarea_individual_2-Realizar_una_aplicación_con_menu/Ejer2/app/src/main/java/com/example/ejer2/MainActivity.java
package com.example.ejer2;

import android.content.Intent;
import android.os.Bundle;


import com.example.ejer2.menu.*;
import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;
import com.example.ejer2.R;
import com.example.ejer2.*;
import com.example.ejer2.menu.Activity1;

import android.util.Log;
import android.view.View;
import android.widget.Button;


public class MainActivity extends AppCompatActivity {

    private Button buttonActivity1, buttonActivity2, buttonActivity3, buttonActivity4;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        buttonActivity1 = findViewById(R.id.buttonActivity1);
        buttonActivity2 = findViewById(R.id.buttonActivity2);
        buttonActivity3 = findViewById(R.id.buttonActivity3);
        buttonActivity4 = findViewById(R.id.buttonActivity4);

        buttonActivity1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, Activity1.class);
                startActivity(intent);
            }
        });

        buttonActivity2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, Activity2.class);
                startActivity(intent);
            }
        });

        buttonActivity3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, Activity3.class);
                startActivity(intent);
            }
        });

        buttonActivity4.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, Activity4.class);
                startActivity(intent);
            }
        });



        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });
    }



}

File: 2DAM/Programacion multimedia y dispositivos moviles/Segundo trimestre/Tarea Individual 8 - Reproduccion audio y video/Ejer8/app/src/main/java/com/example/ejer8/MainActivity.kt
package com.example.ejer8

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        val btnViAct: Button = findViewById(R.id.btnViAct)
        btnViAct.setOnClickListener{

            val intent = Intent(this, videoActivity::class.java)
            startActivity(intent)
        }


        val btnAuAct: Button = findViewById(R.id.btnAuAct)
        btnAuAct.setOnClickListener{

            val intent = Intent(this, audioActivity::class.java)
            startActivity(intent)

        }

    }
}